heralding/capabilities/http.py

```diff
@@ -71,19 +65,18 @@ def do_GET(self):
             headers_bytes = bytes(self.headers['Authorization'], 'utf-8')
             self.wfile.write(headers_bytes)
             self.wfile.write(b'not authenticated')
-        self.request.close()
 
     # Disable logging provided by BaseHTTPServer
     def log_message(self, format_, *args):
         pass
 
 
 class Http(HandlerBase):
-    HandlerClass = BeeHTTPHandler
-
-    def __init__(self, options):
-        super().__init__(options)
+    def __init__(self, options, loop):
+        super().__init__(options, loop)
         self._options = options
 
-    def execute_capability(self, address, gsocket, session):
-        self.HandlerClass(gsocket, address, None, httpsession=session, options=self._options)
\ No newline at end of file
+    async def execute_capability(self, reader, writer, session):
+        http_cap = BeeHTTPHandler(reader, writer, httpsession=session, options=self._options)
+        await http_cap.run()
```

Comment:
We use asyncio.wait_for function (https://docs.python.org/3/library/asyncio-task.html#asyncio.wait_for) in handlerbase.py:
```
await asyncio.wait_for(self.execute_capability(reader, writer, session),
                       timeout=self.timeout, loop=self.loop)
```
First argument of `asyncio.wait_for` must be Future or coroutine object. Therefore each `execute_capability` method (for all capabilities) must be defined using "async def" syntax. 

I use `await http_cap.run()`, because of this: https://github.com/kajojify/heralding/blob/http-telnet/heralding/libs/aiobaserequest.py#L10

In original it looks: https://github.com/python/cpython/blob/master/Lib/socketserver.py#L689
But since `handle()` method is always going to have some awaitable inside (for example `await reader.readline()`), I had to await it  `await handle()`, take out from `__init__` and put to `async def run()`